### PR TITLE
Added Gun Powder recipe - Removed shaped recipe.

### PR DESCRIPTION
**Pull request by dj121739**

### CODE REVIEW

File: src/main/kotlin/net/horizonsend/ion/Ion.kt [hunk 1]
```diff
@@ -83,9 +83,19 @@ class Ion : JavaPlugin() {
...
 		).forEach {
-			val woolType = ShapelessRecipe(NamespacedKey(this, "${it.name.lowercase()}_string_recipe"), ItemStack(Material.STRING, 4))
+			val woolType = ShapelessRecipe(
+				NamespacedKey(this, "${it.name.lowercase()}_string_recipe"),
+				ItemStack(Material.STRING, 4)
+			)
 			woolType.addIngredient(1, it)
 			server.addRecipe(woolType)
+
+			val gunpowderrecipe = ShapelessRecipe(NamespacedKey(this, "gunpowderrecipe"), ItemStack(Material.GUNPOWDER))
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/horizonsend/ion/Ion.kt:93]: gunpowderRecipe

Comment (Peter-Crawley) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt:93]: gunpowder_recipe

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/horizonsend/ion/Ion.kt (hunk 1 above)
Comment (dj121739) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: fixed

Comment (trainb0y1) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: I think what he meant was the val should be `gunpowderRecipe` and the namespaced key should be `gunpowder_recipe`, not both as the latter. (Could be wrong lol, idk)

Comment (Peter-Crawley) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: Correct.

Comment (Peter-Crawley) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: Correct.

Comment (dj121739) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: ah

Comment (dj121739) [reply src/main/kotlin/net/horizonsend/ion/Ion.kt]: fixed again

**Pull request merged**